spec/lib/vmdb/loggers_spec.rb

```diff
@@ -10,12 +10,144 @@ def in_container_env(example)
     ENV['CONTAINER'] = old_env
   end
 
+  shared_examples "has basic logging functionality" do |logger|
+    subject       { logger }
+    let(:logger1) { subject }
+    let(:logger2) { $container_log }
+
+    before do
+      allow($container_log.logdev).to receive(:write)
+    end
+    before(:all) do
+      unless ENV['CONTAINER']
+        logger.extend(ActiveSupport::Logger.broadcast($container_log))
+      end
+    end
+
+    it "responds to #<<" do
+      expect(subject.respond_to?(:<<)).to be true
+    end
+
+    it "responds to #debug" do
+      expect(subject.respond_to?(:debug)).to be true
+    end
+
+    it "responds to #info" do
+      expect(subject.respond_to?(:warn)).to be true
+    end
+
+    it "responds to #error" do
+      expect(subject.respond_to?(:error)).to be true
+    end
+
+    it "responds to #fatal" do
+      expect(subject.respond_to?(:fatal)).to be true
+    end
+
+    it "responds to #unknown" do
+      expect(subject.respond_to?(:unknown)).to be true
+    end
+
+    describe "#datetime_format" do
+      it "works as expected" do
```

Comment:
Don't think you need the outer describe here, though this test is kind of useless?

Reply:
I was more coming from the viewpoint of "if there's only one test in a describe, and the only test has `it 'works'`", then I think it's useless to have the outer describe, when it could just *be* the "it".  But, if the intent was to have more tests, then this is fine.